compositor_render/src/transformations/layout/shader.rs

```diff
@@ -71,19 +68,47 @@ impl LayoutShader {
                     }],
                 });
 
-        let pipeline = pipeline::Pipeline::create_render_pipeline(
-            &wgpu_ctx.device,
-            &pipeline_layout,
-            &shader_module,
-        );
-
-        let empty_texture = Texture::empty(wgpu_ctx);
+        let pipeline = wgpu_ctx
```

Comment:
Any reason why you are not using `pipeline::Pipeline::create_render_pipeline`?  I assumed this code will be the same for all pipelines?